apps/condo/domains/notification/adapters/hcm/messaging.js

```diff
@@ -49,8 +49,10 @@ class HCMMessaging {
         if (!isArray(payload.token)) payload.token = [payload.token]
         if (!isEmpty(payload.data) && !isString(payload.data)) payload.data = JSON.stringify(payload.data)
 
+        const defaultAndroidPayload = cloneDeep(DEFAULT_ANDROID_PAYLOAD)
```

Comment:
Why not just a spread operator?
```suggestion
        const defaultAndroidPayload = { ...DEFAULT_ANDROID_PAYLOAD }
```